.changeset/fuzzy-mangos-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": patch
```

Comment:
Burde det vært en minor-oppgradering for nye features? Eller er dette for smått?

Reply:
consensus on 'minor'?